### PR TITLE
Recycle bin: filter by language

### LINKED ISSUE #204: i18n: Show in /uuid/recycle-bin only entities of current language version

### PR DESCRIPTION
**Pull request by Lodifice**
Partially fixes #204.
This will filter entities and revisions by language, won't work for other kinds of objects.

TODO: add taxonomies

### CODE REVIEW

File: packages/public/server/src/module/Uuid/config/module.config.php [hunk 1]
```diff
@@ -105,6 +105,9 @@
                     'setUuidManager' => [
                         'required' => true,
                     ],
+                    'setInstanceManager' => [
+                        'required' => true,
+                    ]
```
Comment (kulla) [inline packages/public/server/src/module/Uuid/config/module.config.php]: @vroland Run `yarn format:php` after you have finished the work on this PR to format the PHP code and to remove the lint error.

**Comment by inyono** [comment]
What's the state of this PR? Ready to review / merge apart from the missing `yarn format`?

**Comment by Lodifice** [comment]
The code for filtering taxonomies by language is still missing. I thought @vroland would add it soon.

**Comment by vroland** [comment]
@inyono Is there a general way to associate some Item (comment, revision, entity, taxonomy item...) with the language version / instance? It doesn't seem very efficient and maintainable to re-implement filtering for every kind of item.

**Comment by inyono** [comment]
Not that i know of. Especially, since (AFAIK) some items aren't directly tied to the instance / don't have a unique instance. E.g. taxonomy can get its instance via its subject, entities via the taxonomies they are linked to, revisions via their entities, etc.

**Comment by vroland** [comment]
Hm, that's unfortunate. I'll look into how to implement it for taxonomy items when I find the time, but I don't think it makes sense right now to handle every possible item.

**Comment by kulla** [comment]
Yes, when we add support for taxonomy it's enough. We can do this tomorrow at our meeting...

**Pull request merged**